yazi-dds/src/client.rs

```diff
@@ -62,6 +63,50 @@ impl Client {
 		});
 	}
 
+	/// Connect to an existing server and listen in on the messages that are being
+	/// sent by other yazi instances.
+	/// If no server is running, fail right away.
+	/// If a server is closed, attempt to reconnect forever.
+	pub async fn echo_events_to_stdout(kinds: HashSet<String>) -> Result<()> {
+		let mut lines = Self::connect_listener(&kinds).await?;
+
+		loop {
+			match lines.next_line().await {
+				Ok(Some(s)) => {
+					let kind = s.split(',').next();
+					if matches!(kind, Some(kind) if kinds.contains(kind)) {
+						println!("{}", s);
+					}
+				}
+				Ok(None) => loop {
+					println!("Connection closed");
```

Comment:
```suggestion

```

Output not related to the protocol needs to be deleted

Reply:
Done!